### DISCUSSION TITLE
Great! If only I can solve high latency

### DISCUSSION THREAD

**Discussion by ashbeef** [wolveix/satisfactory-server#315] [General]
Oddly, even though my dedicated server is on my own network, I will notice high latency even when playing solo. I will see it dip as low as <20 ms, but then once I start taking game actions, the latency will spike into the (sometimes) multiple hundreds.

I did set SERVERSTREAMING to false as suggested in another thread and I think my single core performance is sufficient (HP EC200a), especially since I only have a couple of other LXC containers running on this host. 

Any additional ideas? I have really tried to research this as much as possible before posting.

**Comment by wolveix** [comment]
Hey @ashbeef! Ensure that your CPU's single core performance is sufficient, also ensure to allocate 12GB+ RAM to your server. Both of these things have fixed the issue for others :)

**Reply by ashbeef** [reply]
Thanks for replying! Unfortunately, I’ve already done both of those things. I mentioned the CPU in my original post but should have said I’m allocating 16 GB of RAM. 

If you have any other ideas I’d love to hear them.

**Reply by wolveix** [reply]
@ashbeef looking up your server, it's a Xeon D-1518 CPU, right? If so, the single core performance is far below what Satisfactory needs unfortunately

**Reply by ashbeef** [reply]
You are totally right - I should have checked this in advance. I'll see what I can make happen with other hardware. Thanks for taking the time! I really appreciate it.
